guides/v2.3/graphql/reference/customer.md

```diff
@@ -117,3 +121,417 @@ The following call returns information about the logged-in customer.
+
+**Manage customers attributes**
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`dob` | String | The customer’s date of birth
+`email` | String | The customer’s email address. Required
+`firstname` | String | The customer’s first name
+`gender` | Int | The customer's gender (Male - 1, Female - 2)
+`is_subscribed` | Boolean | The customer's new password
+`lastname` | String | The customer’s last name
+`middlename` | String | The customer’s middle name
+`password` | String | The customer's password
+`prefix` | String | An honorific, such as Dr., Mr., or Mrs.
+`suffix` | String | A value such as Sr., Jr., or III
+`taxvat` | String | The customer’s Tax/VAT number (for corporate customers)
+
+### Create a customer
+Creates a new customer account.
+
+#### Syntax
+
+`mutation: createCustomer`
+
+#### Example usage
+
+The following call creates a new customer.
+
+**Request**
+
+``` text
+mutation {
+    createCustomer(
+        input: {
+            firstname: "Bob"
+            lastname: "Loblaw"
+            email: "bobloblaw@example.com"
+            password: "b0bl0bl@w"
+            is_subscribed: true
+        }
+    ) {
+        customer {
+            id
+            firstname
+            lastname
+            email
+            is_subscribed
+        }
+    }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "createCustomer": {
+      "customer": {
+        "id": 5,
+        "firstname": "Bob",
+        "lastname": "Loblaw",
+        "email": "bobloblaw@example.com",
+        "is_subscribed": true
+      }
+    }
+  }
+}
+```
+### Update a customer
+
+Updates the customer's personal information.
+
+#### Syntax
+
+`mutation: updateCustomer`
+
+#### Example usage
+
+The following call updates the first and last name and email address for a specific customer.
+
+**Request**
+
+``` text
+mutation {
+    updateCustomer(
+        input: {
+            firstname: "Robert"
+            lastname: "Loblaws"
+            email: "robloblaws@example.com"
+            password: "b0bl0bl@w"
+        }
+    ) {
+        customer {
+            firstname
+            lastname
+            email
+        }
+    }
+}
+```
+
+**Response**
+
+``` json
+{
+  "data": {
+    "updateCustomer": {
+      "customer": {
+        "firstname": "Robert",
+        "lastname": "Loblaws",
+        "email": "robloblaws@example.com"
+      }
+    }
+  }
+}
+```
+
+### Manage customer address {#managecustomeraddress}
+{:.no_toc}
+
+Use these mutations to create or modify the customer's address.
+
+**Manage customer address attibutes**
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`id` | Int | The ID assigned to the address object
+`CustomerAddressInput` | [CustomerAddress](#customerAddress) | An array containing the customer’s shipping and billing addresses
```

Comment:
Unfortunately, the `CustomerAddressInput` object isn't identical to the `CustomerAddress`. You'll need to include the details here.

Reply:
created new table